azurerm/internal/services/compute/tests/resource_arm_virtual_machine_extension_test.go

```diff
@@ -246,6 +274,111 @@ resource "azurerm_virtual_machine" "test" {
   }
 }
 
+resource "azurerm_virtual_machine_extension" "test" {
+  name                 = "acctvme-%d"
+  virtual_machine_id   = "${azurerm_virtual_machine.test.id}"
+  publisher            = "Microsoft.Azure.Extensions"
+  type                 = "CustomScript"
+  type_handler_version = "2.0"
+
+  settings = <<SETTINGS
+	{
+		"commandToExecute": "hostname"
+	}
+SETTINGS
+
+  tags = {
+    environment = "Production"
+  }
+}
+`, data.RandomInteger, data.Locations.Primary, data.RandomInteger, data.RandomInteger, data.RandomInteger, data.RandomInteger, data.RandomInteger, data.RandomInteger, data.RandomInteger)
+}
+
+// TODO: Remove in 2.0
+func testAccAzureRMVirtualMachineExtension_deprecated(data acceptance.TestData) string {
```

Comment:
THis seems to be testing the new property so shouldn't be removed in 2.0/deprecated? 🤔

Reply:
This config uses the deprecated fields (name / location) which won't be available post-2.0, now commented as such.